### PR TITLE
Upgraded dependency Kint from version 3.3 to 4.2.3

### PR DESCRIPTION
**Pull request by Martin-4Spaces**
https://github.com/podio-community/podio-php/issues/221

### CODE REVIEW

**Review by daniel-sc — COMMENTED** [review]
Could you please check the one comment of mine? Besides that it LGTM - I'll have a look at the failed build..

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   ],
   "require": {
     "php": "^7.3 || ^8.0",
-    "kint-php/kint": "^3.2",
+    "kint-php/kint": "^4.2",
```
[the remark this answers is not in the document]
Comment (Martin-4Spaces) [reply composer.json:26]: Usually we want patches to get through. 
But I can change it if you want? :)

[the remark this answers is not in the document]
Comment (daniel-sc) [reply composer.json:26]: Ok - now you probably know more than me 😊 to me 4.2.0 looks cleaner, as it is immediately clear that the package uses MAJOR.MINOR.PATCH versioning - but if 4.2 is equivalent it is fine for me as well.

[the remark this answers is not in the document]
Comment (Martin-4Spaces) [reply composer.json]: Done :)

**Review by daniel-sc — APPROVED** [review]

**Comment by Martin-4Spaces** [comment]
@daniel-sc What is missing for this PR to be merged into master?

**Comment by daniel-sc** [comment]
@Martin-4Spaces we need to repair the build pipeline - do you want to take a look? (my first research indicates it is some non-trivial problem with runner images and/or action versions..)
I'll look into it later or tomorrow if I find time - in case I find something I'll post here or update + merge the pr..

**Comment by Martin-4Spaces** [comment]
Awesome job @daniel-sc!

**Pull request merged**